source/i18n/zh-CN.ts

```diff
@@ -1,4 +1,25 @@
 export default {
     review: '回顾',
-    register: '注册'
+    register: '注册',
+    aboutus: '关于我们',
+    contactus: '联系我们',
+    homePagetitle: '成都 Web 开发者大会',
```

Comment:
```suggestion
    chengdu_web_developer_summit: '成都 Web 开发者大会',
```
左边键名要和右边文本内容对应，除非文本过长时简述。